docker-compose.yml

```diff
@@ -14,14 +14,16 @@ services:
     environment:
       - NEAR_RPC_URL=https://rpc.testnet.nearprotocol.com
       #- NEAR_RPC_URL=https://rpc.betanet.nearprotocol.com
+      - NEAR_GENESIS_RECORDS_URL=https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/testnet/genesis.json
       - WAMP_NEAR_EXPLORER_URL=ws://wamp:8080/ws
       - WAMP_NEAR_EXPLORER_BACKEND_SECRET=back
+    mem_limit: 500M
 
-  frontend:
-    build: ./frontend
-    environment:
-      - PORT=3000
-      - WAMP_NEAR_EXPLORER_URL=ws://localhost:8080/ws
-      - WAMP_NEAR_EXPLORER_INTERNAL_URL=ws://wamp:8080/ws
-    ports:
-      - 3000:3000
+  # frontend:
+  #   build: ./frontend
+  #   environment:
+  #     - PORT=3000
+  #     - WAMP_NEAR_EXPLORER_URL=ws://localhost:8080/ws
+  #     - WAMP_NEAR_EXPLORER_INTERNAL_URL=ws://wamp:8080/ws
+  #   ports:
+  #     - 3000:3000
```

Comment:
```suggestion
  frontend:
    build: ./frontend
    environment:
      - PORT=3000
      - WAMP_NEAR_EXPLORER_URL=ws://localhost:8080/ws
      - WAMP_NEAR_EXPLORER_INTERNAL_URL=ws://wamp:8080/ws
    ports:
      - 3000:3000
```

Reply:
I am still figure out memory problem and after all the fixes, I will return docker file to original one.